bundles/binding/org.openhab.binding.knx/src/main/java/org/openhab/binding/knx/internal/dpt/KNXCoreTypeMapper.java

```diff
@@ -64,179 +74,473 @@
  *
  * @author Kai Kreuzer
  * @author Volker Daube
- * @author Jan N. Klug
+ * @author Helmut Lehmeyer - Java8, calimero 2.4, generic DPT Mapper
```

Comment:
Calimero 2.4? I don't see it. Is it coming soon?

Reply:
done.